cinema/serializers.py

```diff
@@ -18,6 +18,59 @@ def update(self, instance, validated_data):
             "description", instance.description
         )
         instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+
+        return instance
+
+
+class CinemaHallSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=25)
```

Comment:
If you don't include the id field, the API responses for actors will not show the id, and you won't be able to use it for updates or references in related models (like for associating an actor with a movie). This can be problematic if you're dealing with many-to-many relationships or need to update or delete specific actor records.
Add id field in every serializer